### PR TITLE
[Core] Merge cucumber-html into cucumber-core

### PR DESCRIPTION
**Pull request by grasshopper7**
## Summary

Copied HtmlFormatter resources from cucumber-html to cucumber-core. 

Fixes: #1647 

## Details

1. Copied resources from cucumber/formatter in cucumber-html into io/cucumber/formatter/html in cucumber-core.
2. Changed resource paths in HTMLFormater.
3. Removed cucumber-html dependency from cucumber-core pom.xml.
4. Added /some directory in gitignore to allow report.js to be tracked.

## Motivation and Context

Refer [issue](https://github.com/cucumber/cucumber-jvm/issues/1647)

## How Has This Been Tested?

Passes existing test in HTMLFormatterTest, no additional functionality added.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).

## Checklist:

- [ ] I've added tests for my code.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,9 +14,10 @@ target/
 tmp/
 gen-external-apklibs/
 out/
+some/
```
Comment (mpkorstanje) [inline .gitignore]: We should really figure out who creates this directory.

Comment (grasshopper7) [reply .gitignore]: HTMLFormatterTest is the culprit.

Comment (mpkorstanje) [reply .gitignore]: I'll make a seperate PR to fix that.

File: .gitignore [hunk 2]
```diff
@@ -14,9 +14,10 @@ target/
...
 
 # Build & test droppings
-report.js
+# report.js
```
Comment (mpkorstanje) [inline .gitignore]:
```suggestion
```

File: .gitignore (hunk 1 above)
Comment (mpkorstanje) [inline .gitignore]:
```suggestion
```

**Comment by mpkorstanje** [comment]
@grasshopper7 could you rebase your branch on master? I've made sure that the tests will write their files to the `target` folder now. So the changes to `.gitignore` aren't needed anymore.

**Comment by mpkorstanje** [comment]
Owh cool I can do that online too.

**Pull request merged**